src/Creator/RequestCommandCreatorInterface.php

```diff
@@ -13,10 +13,11 @@
 
 namespace Sylius\RefundPlugin\Creator;
 
-use Sylius\RefundPlugin\Command\RefundUnits;
 use Symfony\Component\HttpFoundation\Request;
 
-interface RefundUnitsCommandCreatorInterface
+interface RequestCommandCreatorInterface
 {
-    public function fromRequest(Request $request): RefundUnits;
+    public function fromRequest(Request $request): object;
 }
+
+class_alias(RequestCommandCreatorInterface::class, 'Sylius\RefundPlugin\Creator\RefundUnitsCommandCreatorInterface');
```

Comment:
nice 👌 

but If I remember correctly the previously existing class needs to stay where it was, to satisfy static analysers etc.